examples/acceptance_tests/025/lib/test.ak

```diff
@@ -1,3 +1,6 @@
+// Should this fail or equal true? 
+// Should != be false or true?
```

Comment:
The test should pass. Nil is also a terrible name as we pointed out in the past; this is really unit. And unit is equal to itself. It's merely a 0-ary constructor. 

I think we should also change the notation to `#()` perhaps. cc @rvcas